test/common/include/alpaka/test/event/EventHostManualTrigger.hpp

```diff
@@ -343,3 +343,346 @@ namespace alpaka
+                    //   on host updates may hang. This includes synchronization between the host and
+                    //   the device build upon value-based CUDA stream synchronization APIs such as
+                    //   cuStreamWaitValue32() and cuStreamWriteValue32().
+                    ALPAKA_CUDA_DRV_CHECK(
+                        cuStreamWaitValue32(
+                            (CUstream)stream.m_spStreamCudaRtAsyncImpl->m_CudaStream,
+                            (CUdeviceptr)event.m_spEventImpl->m_devMem,
+                            0u,
+                            CU_STREAM_WAIT_VALUE_GEQ));
+                }
+            };
+            //#############################################################################
+            //!
+            //#############################################################################
+            template<>
+            struct Enqueue<
+                stream::StreamCudaRtSync,
+                test::event::EventHostManualTriggerCuda>
+            {
+                //-----------------------------------------------------------------------------
+                //
+                //-----------------------------------------------------------------------------
+                ALPAKA_FN_HOST static auto enqueue(
+                    stream::StreamCudaRtSync & stream,
+                    test::event::EventHostManualTriggerCuda & event)
+                -> void
+                {
+                    ALPAKA_DEBUG_MINIMAL_LOG_SCOPE;
+
+                    // Copy the shared pointer to ensure that the event implementation is alive as long as it is enqueued.
+                    auto spEventImpl(event.m_spEventImpl);
+
+                    // Setting the event state and enqueuing it has to be atomic.
+                    std::lock_guard<std::mutex> lk(spEventImpl->m_mutex);
+
+                    // The event should not yet be enqueued.
+                    assert(spEventImpl->m_bIsReady);
+
+                    // Set its state to enqueued.
+                    spEventImpl->m_bIsReady = false;
+
+                    // PGI Profiler`s User Guide:
+                    // The following are known issues related to Events and Metrics:
+                    // * In event or metric profiling, kernel launches are blocking. Thus kernels waiting
+                    //   on host updates may hang. This includes synchronization between the host and
+                    //   the device build upon value-based CUDA stream synchronization APIs such as
+                    //   cuStreamWaitValue32() and cuStreamWriteValue32().
+                    ALPAKA_CUDA_DRV_CHECK(
+                        cuStreamWaitValue32(
+                            (CUstream)stream.m_spStreamCudaRtSyncImpl->m_CudaStream,
+                            (CUdeviceptr)event.m_spEventImpl->m_devMem,
+                            0u,
```

Comment:
IMO this must be `16843009u`

Reply:
This is correct... I was wrongly thinking that memset sets a 32 bit value ;-)

Reply:
`0x01010101u` should be more idiomatic